options/locale/locale_en-US.ini

```diff
@@ -1055,6 +1055,8 @@ migrate.migrating_labels = Migrating Labels
 migrate.migrating_releases = Migrating Releases
 migrate.migrating_issues = Migrating Issues
 migrate.migrating_pulls = Migrating Pull Requests
+migrate.cancel_migrating_title = Cancel Migrating
```

Comment:
```suggestion
migrate.cancel_migrating_title = Cancel Migration
```

Reply:
I think `Migrating` is better, because it means "cancel this operation (migrating)"

`Migration` could also be fine, I am more or less netural.

Reply:
I'm not a native English speaker but `Migration` sounded better to me